test-requirements.txt

```diff
@@ -2,36 +2,23 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --upgrade --output-file test-requirements.txt test-requirements.in
+#    pip-compile --output-file=test-requirements.txt test-requirements.in
```

Comment:
Urf, I wish `pip-compile` would actually look at its arguments before formatting this line in here... It's wrong now.